roles/iridl/templates/update_datalib

```diff
@@ -84,6 +84,9 @@ init_git {{python_maproom_repo.url}} {{python_maproom_src_dir}} \
          {{python_maproom_repo.version}} {{git_ssh_prefix}}{{python_maproom_repo.name}}
 
 docker build -t {{python_maproom_repo.name}}:{{python_maproom_repo.version}} . || fail $? "python maproom build failed"
+
+cd {{python_maproom_src_dir}}/update || fail $? "python maproom repo does not have an update script"
```

Comment:
Maprooms with small, application-specific datasets might include their data in the container and not need an update script. Do we force them to provide a no-op Dockerfile and update script anyway, just to satisfy the interface, or do we make the update container optional?

Reply:
Only build the update script container if the update subdir exists in the repo